lib/exzeitable.ex

```diff
@@ -63,7 +60,12 @@ defmodule Exzeitable do
       end
 
       def mount(_map, assigns, socket) do
-        assigns = %{params: Params.new([], unquote(opts), __MODULE__)}
+        param_assigns = Keyword.get(unquote(opts), :assigns, %{})
```

Comment:
we've moved this file into Pangea , so while this change still applies, this file isn't used anymore